### PR TITLE
Snapshot release v5

### PR DESCRIPTION
**Pull request by twifkak**

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
done

File: go.mod [hunk 1]
```diff
@@ -7,16 +7,18 @@ require (
...
+	github.com/kylelemons/godebug v1.1.0
 	github.com/pelletier/go-toml v1.1.0
 	github.com/pkg/errors v0.8.1
 	github.com/pquerna/cachecontrol v0.0.0-20180306154005-525d0eb5f91d
 	github.com/prometheus/client_golang v1.1.0
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
 	golang.org/x/net v0.0.0-20190930134127-c5a3c61f89f3
-	google.golang.org/grpc v1.20.1
+	google.golang.org/grpc v1.27.0
```
Comment (banaag) [inline go.mod:20]: Just to double check, did you try building the amppkg with -tags dns01? Just to make sure these library upgrades didn't break that feature.

**Comment by Gregable** [comment]
release signed

**Review by Gregable — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (twifkak) [reply go.mod:20]: Yes, it builds. Though, when running I get this on stdout:

```
WARNING: Package "github.com/golang/protobuf/protoc-gen-go/generator" is deprecated.
        A future release of golang/protobuf will delete this package,
        which has long been excluded from the compatibility promise.
```

**Review by banaag — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (twifkak) [reply go.mod:20]: Filed #482. I don't think it's a release blocker.

**Comment by twifkak** [comment]
Thanks all!

**Pull request merged**